### PR TITLE
NEVISACCESSAPP-6212: update gh actions

### PR DESCRIPTION
**Pull request by daniel-toth-leeder**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -67,5 +67,5 @@ flutter {
 
 dependencies {
     //noinspection GradleDynamicVersion
-    debugImplementation 'ch.nevis:nevis-mobile-authentication-sdk-android-debug:3.7.+'
+    debugImplementation 'ch.nevis:nevis-mobile-authentication-sdk-android-debug:3.8.+'
```
Comment (tamas-toth) [inline android/app/build.gradle:70]: 🔧 Please do not merge this into main, it is still unreleased :)

File: ios/Podfile [hunk 2]
```diff
@@ -33,8 +33,8 @@ target 'Runner' do
 	use_frameworks!
 	use_modular_headers!
 
-	pod 'NevisMobileAuthentication-Debug', '~> 3.7.0', :configurations => ['Debug', 'Profile']
-	pod 'NevisMobileAuthentication', '~> 3.7.0', :configurations => ['Release']
+	pod 'NevisMobileAuthentication-Debug', '~> 3.8.0', :configurations => ['Debug', 'Profile']
+	pod 'NevisMobileAuthentication', '~> 3.8.0', :configurations => ['Release']
```
Comment (tamas-toth) [inline ios/Podfile:37]: 🔧 Please do not merge this into main, it is still unreleased :)

File: pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: nevis_mobile_authentication_sdk_example_app_flutter
 description: A new Flutter project.
-version: 3.7.0+1
+version: 3.8.0+1
```
Comment (tamas-toth) [inline pubspec.yaml:3]: 🔧 Please do not merge this into main, it is still unreleased :)

File: pubspec.yaml [hunk 4]
```diff
@@ -35,7 +35,7 @@ dependencies:
   dio_cookie_manager: ^3.1.1
   cookie_jar: ^4.0.8
   permission_handler: ^11.3.1
-  nevis_mobile_authentication_sdk: '>=3.7.0 <3.8.0'
+  nevis_mobile_authentication_sdk: '>=3.8.0 <3.9.0'
```
Comment (tamas-toth) [inline pubspec.yaml:38]: 🔧 Please do not merge this into main, it is still unreleased :)

**Review by tamas-toth — CHANGES_REQUESTED** [review]

**Review by viktor-gulyas — APPROVED** [review]

**Pull request merged**